crud_handler/crud_handler.go

```diff
@@ -0,0 +1,191 @@
+package crud_handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"main/transformer"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+)
+
+type TransformRequest struct {
+	Type        string `json:"type"`
+	CaesarShift int    `json:"shift,omitempty"`
+	Input       string `json:"input,omitempty"`
+}
+
+type TransformResult struct {
+	Id          string `json:"uuid"`
+	Type        string `json:"type"`
+	CaesarShift int    `json:"shift,omitempty"`
+	Result      string `json:"result,omitempty"`
+	Created_at  int64  `json:"created"`
```

Comment:
don't use underscore, use `time.Time` for time representation, rename to `created_at` in json tag